include/boost/compute/memory_object.hpp

```diff
@@ -16,6 +16,7 @@
 #include <boost/compute/kernel.hpp>
 #include <boost/compute/detail/get_object_info.hpp>
 #include <boost/compute/detail/assert_cl_success.hpp>
+#include <boost/bind.hpp>
```

Comment:
Hmm, I don't think this `#include` is necessary (looks like it's just used by the test)?